src/js/pages/bike-count/bike-count-per-year-chart.js

```diff
@@ -23,7 +23,7 @@ const years = [
   2017
 ];
 // For drawing the lines
-const count = [
+let count = [
```

Comment:
Why those data do not come from the API ?

Reply:
It's coming from the API. This variable is used as template for the dataset